### PR TITLE
Switch to using makara2 to show off specialization

### PR DESCRIPTION
**Pull request by grawk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,55 +131,62 @@ What above config means is for eg: partial `yingyang` will be replaced with part
...
-* Modify `config/config.json` to load in these rules under the property `specialization`
-
-```
-"specialization": "import:./specialization.json"
-```
-
 ### Including specialization into the render work flow
-#### Modify `view engines` in `config/config.json`
+
+#### Modify `view engines` in `config/config.json` to use `makara@2` to use `makara@2` to use `makara@2` to use `makara@2`
```
Comment (aredridel) [inline README.md]: +#### Modify `view engines` in `config/config.json` to use `makara@2` to use `makara@2` to use `makara@2` to use `makara@2`

**Comment by aredridel** [comment]
Looking good, that one vimo aside.

**Comment by grawk** [comment]
Looks like the makara2 branch in this repo is identical to the makara2 branch in your fork @aredridel shall I close the other PR?

**Comment by aredridel** [comment]
Sure!

**Comment by grawk** [comment]
Going to have to resolve this first: https://github.com/krakenjs/engine-munger/issues/45

Given the fact that (as far as I know) specialization is not used, it may be a while before issue 45 is resolve. I'll need to move on to higher priority items for now.

**Comment by grawk** [comment]
https://github.com/krakenjs/engine-munger/issues/45 is now resolved so we can merge this.

**Comment by totherik** [comment]
:+1: All the red makes me happy.

**Comment by grawk** [comment]
making it so

**Pull request merged**